web-jehlomat/src/Components/Inputs/DateTimeInput/DateInput.tsx

```diff
@@ -0,0 +1,89 @@
+import { FC, InputHTMLAttributes, useState } from 'react';
+import { default as MInput } from '@mui/material/TextField';
+import styled from '@emotion/styled';
+import DateAdapter from '@mui/lab/AdapterDayjs';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import MobileDatePicker from '@mui/lab/MobileDatePicker';
+
+interface Props extends InputHTMLAttributes<HTMLInputElement> {
+    name?: string;
+    label?: string;
+    error?: string | undefined;
+    fullWidth?: boolean;
```

Comment:
I would suggest to remove `fullWidth` prop, because one can use `width: "100%"` instead.